modules/lang-painless/src/main/java/org/elasticsearch/painless/action/PainlessContextClassBindingInfo.java

```diff
@@ -66,11 +66,11 @@
         PARSER.declareStringArray(ConstructingObjectParser.constructorArg(), PARAMETERS);
     }
 
-    private final String declaring;
-    private final String name;
-    private final String rtn;
-    private final int readOnly;
-    private final List<String> parameters;
+    public final String declaring;
```

Comment:
Its probably worth a comment about why these are public instead. It'd be nice if we didn't need it though!

Reply:
Would you prefer these be getters instead or comments here?

Reply:
Hmmmm. I think getters would be more "normal java" to be honest. Personally, I think `public final` is fine but I think we're trying to be more "normal java".... :man_shrugging: